tap-gui/plugins/runtime-resource-visibility.hbs.md

```diff
@@ -138,12 +134,21 @@ Users can see the pod table in each resource details page.
 
 ### <a id="pod-details-metrics"></a> Overview of pod metrics
 
-The overview card displays the user-configured resource limits on the pod, defined in accordance with
-the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/).
-These limits do not represent actual real-time resource use. To monitor actual real-time resource use,
-see [Application Live View for Spring Boot Applications in Tanzu Application Platform GUI](app-live-view-springboot.md).
+If you have a metric server running on your cluster, you will find that the
```

Comment:
Suggest removing "you will find", also add "s" to "metric":

```
If you have a metrics server running on your cluster, the overview card displays realtime metrics for pods.
```

Reply:
Addressed, can you please re-review it? @menehune23

Reply:
Addressed, please consider resolved.